Cargo.toml

```diff
@@ -130,6 +130,7 @@ cargo-args = ["-Zbuild-std=core"]
 
 [dependencies]
 rustc-std-workspace-core = { version = "1.0.0", optional = true }
+zerocopy = { version = "0.7.16", optional = true, features = ["derive"] }
```

Comment:
Leaving a TODO for myself:
- [ ] Rename to `zerocopy-0-7`
- [ ] Add `zerocopy-0-6` (to be removed before merge) to test that multiple version trains work

Since libc doesn't specify an edition, we have to use the legacy approach of importing macros via `#[macro_use] extern crate zerocopy;`, which may not play nicely with having macros from multiple versions of the same crate. Instead, we may need to update `build.rs` to detect when the Rust version is early enough that editions are supported, and instruct rustc to use a recent-enough version that we can explicitly import different versions under different names. This would be necessary in case a build requires multiple zerocopy versions simultaneously.

Reply:
See for more discussion: https://github.com/google/zerocopy/issues/557

Reply:
Once https://github.com/rust-lang/libs-team/issues/72 gets a conclusion, we can specify the edition.